CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+ * **Be surgical:** What is the smallest diff you can make that still accomplishes your goal?
+ Avoid the urge to fix unrelated stylistic issues in a file you are changing.
+ * **Write tests first:** If you're changing a specific part of the codebase, first ensure it has good test coverage; if it does, write a few more tests yourself anyway!
+ Writing tests is a great way to learn how the code functions.
+ If possible, submit your tests in a separate PR so their benefits can be realized immediately.
+ We don't fully subscribe to Test-Driven Development (TDD) but having good existing test coverage of a changed area is a prerequisite to changes being merged.
+ * **Split up your changes:** If parts of your changes provide some benefit by themselves - like additional tests - split them into a separate PR.
+ It is preferable to have several small PRs merged quickly than one large PR that takes longer to review.
+ * **Craft your commits well:** This can require advanced git knowledge (see [DEVELOPING.md](DEVELOPING.md)).
+ Treat your commits as forms of communication, not a way to bludgeon the codebase into shape.
+ You don't need to check this, but ideally the build & tests should pass on every commit.
+ This means you will often be amending commits as you work instead of adding new ones.
+
+When you open a PR against this repo, the continuous integration (CI) checks will run.
+These ensure your changes do not break the build & tests.
+While you can run most of these checks locally, if you'd like to run the CI at any time you can open a PR between your feature branch and the master branch of *your own* fork of this repo.
+
+Contribution Possibilities
+--------------------------
+
+These tools have a large number existing issues to fix, which you can see on the repo's [issues tab](https://github.com/tlaplus/tlapm/issues).
+
+Non-code contributions are particularly desired!
+TLAPM documentation needs a lot of work.
+The docs are hosted at https://proofs.tlapl.us and live in this repo in the [doc/web](doc/web) directory.
+
+For new features, there are a number of [substantial improvements we'd like to see implemented](todo.txt).
+If you have an idea for a new feature, open an issue in this repository to start a discussion.
+For more fundamental proposals that involve changes to the TLA⁺ language itself, you can [open a RFC](https://github.com/tlaplus/rfcs/issues).
+Often it will be necessary to build community support for your idea; for this you can use the [mailing list](https://groups.google.com/g/tlaplus) and especially the [monthly online community meeting](https://groups.google.com/g/tlaplus/c/CpAEnrf-DHQ/m/YrORpIfSBwAJ).
+Expressions of goodwill like volunteering labor to review pending PRs is also appreciated!
+
+Getting Started
+---------------
+
+Take the time to set up your local development environment.
+See [DEVELOPING.md](DEVELOPING.md) for details.
+Good luck, and have fun!
```

Comment:
Why "Good luck", is this standard in CONTRIBUTING? Perhaps "Have fun, and keep in touch!"?